test/helpers/node.go

```diff
@@ -149,21 +149,29 @@ type ExecOptions struct {
 	SkipLog bool
 }
 
-// Exec returns the results of executing the provided cmd via SSH.
-func (s *SSHMeta) Exec(cmd string, options ...ExecOptions) *CmdRes {
+// ExecWithParams returns the results of executing the provided cmd with the
+// given cmd parameters via SSH.
+func (s *SSHMeta) ExecWithParams(cmd string, cmdParams []string, options ...ExecOptions) *CmdRes {
 	var ops ExecOptions
 	if len(options) > 0 {
 		ops = options[0]
 	}
+	cmdStr := cmd
```

Comment:
I would not do like this, and I would follow ginkgo standar. 

https://github.com/cilium/cilium/blob/0c9728feab4f3354634e8db07512ac4b72a66137/test/helpers/wrappers.go#L52-L54

Reply:
@eloycoto that might break if the parameters contain spaces. e.g. `$ cat file with spaces`.

Reply:
Could you give an example? 

The function will be like: 
`ExecWithParams("cat %s %s %d %q", []string{a,b,c,d})`

I don't see how it can be broken.